src/bootstrap-table.js

```diff
@@ -1336,6 +1340,7 @@ class BootstrapTable {
       Utils.sprintf(' %s', htmlAttributes.length ? htmlAttributes.join(' ') : undefined),
       Utils.sprintf(' id="%s"', Array.isArray(item) ? undefined : item._id),
       Utils.sprintf(' class="%s"', style.classes || (Array.isArray(item) ? undefined : item._class)),
+      Utils.sprintf(' style="%s"', Array.isArray(item) ? undefined : item._style),
```

Comment:
Im not sure what happens if you apply `undefined` on the `sprintf` function.
Could you check that ?

Reply:
It returns empty when the parameters are undefined https://jsfiddle.net/djhvscf/xv0f6cyd/

Reply:
So it would make more sense to check it with a if to prevent empty style attributes (`style=""`).

Reply:
It's OK: https://github.com/wenzhixin/bootstrap-table/blob/develop/src/utils/index.js#L2

Reply:
oh okay, i thought %s would be replaced with an empty string :D